docs/webhooks/etl-pipelines.mdx

```diff
@@ -141,3 +143,33 @@ When you have completed the setup, please provide Vital the following informatio
 
 - An `ampqs://` connection string with username and password embedded
 - Whether or not you want larger blobs (≥ 1KiB) to be gzipped.
+
+## Azure Event Hub
+
+### Message Structure
+
+Each message would be published by Vital message queue with the following attributes:
+
+| Type | Key | Value |
+| --- | --- | --- |
+| Attribute | event_type | Vital event type, e.g., daily.data.activity.created |
+| Attribute | blob_type | json: UTF8 JSON document |
```

Comment:
These do not seem to be reflective of what we sent. Azure has custom message headers with HTTP-like scheme instead of attributes for one.